### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nithasha-samad**
Hello!

* Type: bug fix | new feature | code quality | documentation
* Link to issue:

**In raising this pull request, I confirm the following:**

- [ ] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [ ] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [ ] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,31 +6,31 @@
...
 
 ## Installation
 
-For detailed installation instructions you can check our [installation](https://docs.phalcon.io/4.0/en/installation) page in the docs.
+For detailed installation instructions, you can check our [installation](https://docs.phalcon.io/4.0/en/installation) page in the docs.
 
 ## Generating API Documentation
 
-Generating new documentation files for docs repository can be done using the script in tests/_ci/generate-api-docs.php.
+Generating new documentation files for docs repository can be done using the script in tests/_ci/generate-API-docs.php.
```
Comment (niden) [inline README.md]: This is wrong. `API` and `api` are not the same in Linux/Mac environments

File: README.md [hunk 2]
```diff
@@ -6,31 +6,31 @@
...
 Steps:
 - Clone the phalcon repo
-- Checkout the tag you would like to generate docs for.
-- Run `php tests/_ci/generate-api-docs.php`
+- Check out the tag you would like to generate docs for.
+- Run `PHP tests/_ci/generate-API-docs.php`
```
Comment (niden) [inline README.md]: Same here. `php` is the command not uppercase. Same with `API`

File: README.md [hunk 3]
```diff
@@ -76,4 +76,4 @@ Phalcon is open source software licensed under the BSD 3-Clause License.
 
 Copyright © 2011-present, Phalcon Team.
 
-See the [LICENSE.txt](https://github.com/phalcon/cphalcon/blob/master/LICENSE.txt) file for more. Additional licenses of packages that Phalcon uses, is inspired by or has adapted is located in the [3rdparty/licenses][https://github.com/phalcon/cphalcon/blob/master/3rdparty/licenses] directory.
+See the [LICENSE.txt](https://github.com/phalcon/cphalcon/blob/master/LICENSE.txt) file for more. Additional licenses of packages that Phalcon uses are inspired by or have adapted is located in the [3rdparty/licenses][https://github.com/phalcon/cphalcon/blob/master/3rdparty/licenses] directory.
```
Comment (niden) [inline README.md]: Please revert this. 

Explanation: Additional licenses that Phalcon uses, is inspired by (Phalcon that is), or has adapted (again Phalcon has adapted)

File: README.md [hunk 4]
```diff
@@ -6,31 +6,31 @@
...
 
 A big thank you to [our Backers](https://github.com/phalcon/cphalcon/blob/master/BACKERS.md); you rock!
 
 ## Getting Started
 
 Phalcon is written in [Zephir/C](https://zephir-lang.com/) with platform independence in mind.
-As a result, Phalcon is available on Microsoft Windows, GNU/Linux, FreeBSD and MacOS.
-You can either download a binary package for the system of your choice or build it from source.
+As a result, Phalcon is available on Microsoft Windows, GNU/Linux, FreeBSD, and macOS.
+You can either download a binary package for the system of your choice or build it from the source.
```
Comment (niden) [inline README.md]: Please remove the `the`. Build from source is correct.

**Review by niden — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (CameronHall) [reply README.md]: Should read
> Additional licenses of packages that Phalcon uses inspired by or adapted **are** located in the [3rdparty/licenses][https://github.com/phalcon/cphalcon/blob/master/3rdparty/licenses] directory.

Also if we used LICENSE.md we'd get the pretty license tldr from GitHub as well.

File: README.md (hunk 2 above)
Comment (virgofx) [reply README.md]: Yep agree with @niden on this. Second line needs to be all lowercase

**Comment by ruudboon** [comment]
@niden I think it will be easier if we this add this to the 4.0.x before RC.2

**Comment by niden** [comment]
We can cherry pick it after the changes are made.

**Comment by niden** [comment]
Changes requested have been implemented here: https://github.com/phalcon/cphalcon/pull/14536

**Comment by niden** [comment]
Thank you @nithasha-samad

**Pull request closed without merging**